project/utils/utils.py

```diff
@@ -156,3 +156,45 @@ def print_config(
 
     # with open("config_tree.log", "w") as file:
     #     rich.print(tree, file=file)
+
+
+## HF datamodule specific
+
+
+def get_task_info(task_name):
```

Comment:
A type hint would be nice, as it will be used by the auto-schema script and show the possible values in the datamodule config (assuming you also use it to annotate the constructor argument of the datamodule).

This should probably be moved to the same file as the  HF datamodule.

```suggestion
SupportedTask = Literal["cola", "sst2", "mrpc", "qqp", "stsb", "mnli", "qnli", "rte", "wnli", "ax"]
def get_task_info(task_name: SupportedTask):
```

Reply:
fixed by 1e989128562678b549fd48ac847119d0aad52311

Reply:
Could also use the type in the constructor of the datamodule, so the auto_schema.py script will use it when contructing the schema for the config